src/data_collectors.js

```diff
@@ -177,6 +295,22 @@ function collectCommonNavData (dialectsData) {
   return commonNavData
 }
 
+function sortProps (props) {
```

Comment:
This belongs to `utils.js`. Also sorting function from line 304 is already implemented in `utils.nameSorter`.